### ISSUE TITLE
[Bug]: DHX mining income list readability

### ISSUE THREAD

**Issue by havadevs** [mxc-foundation/supernode-app#299]
### Contact Details [Discord Username]

jibo

### What happened?

The dark blue circle against the enclosed black number makes the number difficult to read.

![Screenshot_20210723-215728](https://user-images.githubusercontent.com/66387291/126836082-a2d3ea3b-b669-4d59-a838-a8aaa99cf54b.jpg)

### Version

1.13.0 (Beta)

### Which OS are you seeing the problem on?

Android

**Comment by jeffstahlnecker** [comment]
@havadevs --- was this for 1.13.1? Just realised it wasn't an option in the form.

**Comment by havadevs** [comment]
> @havadevs --- was this for 1.13.1? Just realised it wasn't an option in the form.

No, this is still 13.0. Im waiting for the playstore beta13.1 to drop. Cheers!

**Comment by jeffstahlnecker** [comment]
@havadevs should be ready now - can you check?

**Comment by havadevs** [comment]
> @havadevs should be ready now - can you check?

ok bug #300 resolved on v13.1 It can be closed

**Comment by havadevs** [comment]
> @havadevs should be ready now - can you check?

bug #299 is still in 1.13.2

**Comment by havadevs** [comment]
Note that mxc staked assets list also has the dark blue issue.

![Screenshot_20210801-095814.jpg](https://user-images.githubusercontent.com/66387291/127763877-daac020e-892c-44ee-a3c4-d9dd15263a12.jpg)

**Comment by rd101rahul** [comment]
This issue has been addressed and the fix would be available with next build.
